### DISCUSSION TITLE
Does SIGMA allow for sequence

### DISCUSSION THREAD

**Discussion by austinsonger** [SigmaHQ/sigma#2160] [Q&A] [answered]
Does SIGMA allow for the creation of detection rules that have a defined sequence? Kinda like how Elastic EQL allows for sequence detections?

**Comment by Neo23x0** [comment, marked as answer]
The new converter that supports correlation rules can do that. 
https://twitter.com/cyb3rops/status/1338180505511063554?lang=en

However, it always depends on the respective backend if it supports such queries.
